lib/core/presentation/widgets/chat/matrix_avatar.dart

```diff
@@ -48,33 +54,93 @@ class MatrixAvatar extends StatelessWidget {
         ),
       ),
     );
-    final borderRadius = BorderRadius.circular(radius ?? LemonRadius.extraSmall);
-    final container = ClipRRect(
-      borderRadius: borderRadius,
-      child: Container(
-        width: size,
-        height: size,
-        color: noPic
-            ? name?.lightColorAvatar
-            : Theme.of(context).secondaryHeaderColor,
-        child: noPic
-            ? textWidget
-            : MxcImage(
-                key: Key(mxContent.toString()),
-                uri: mxContent,
-                fit: BoxFit.cover,
-                width: size,
-                height: size,
-                placeholder: (_) => textWidget,
-                cacheKey: mxContent.toString(),
-              ),
-      ),
+    final borderRadius = BorderRadius.circular(radius ?? LemonRadius.xSmall);
+    final container = Stack(
+      alignment: Alignment.center,
+      children: [
```

Comment:
- Avoid putting too many variables in build method. 
- For rich widget such as `container` here I suggest we put it in separate class. 
- We don't need an if else statement whether onTap or null.

Reply:
> We don't need an if else statement whether onTap or null.

Thanks, remove the weird condition here

> Avoid putting too many variables in build method.
> For rich widget such as container here I suggest we put it in separate class.

This widget not so big, it's just render MatrixAvatar only